src/App.js

```diff
@@ -1,14 +1,19 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
 import { EndPointContainer }  from './EndPointContainer.js'
 
 class App extends Component {
-  render() {
+  render() { 
     return (
-      <div className="App">
-        <header className="App-header">
+      <div className="app-container">
+        <header className="doc-header">
+        <span className="logo-container">
+          <h1 className="logo-text">Taco Restaurants API</h1>
+        </span>
+          <h3 className="header-subtext">Find Top Tacos Across the US!</h3>
         </header>
+        <div className="taco-image-container">
+        </div>
```

Comment:
i'm sure we're going to end up with merge conflicts, but they shouldn't be too hard to work out